### PR TITLE
Updated the default files to match the starting project files. fixes #2079

### LINKED ISSUE #2078: New default HTML doesn't match HTML file we add from Files icon
different-html

We changed the default HTML and CSS in https://github.com/mozilla/thimble.mozilla.org/commit/4b241cdbc69ca69e73fbe8700433d632a9c02520 but didn't update the ones we inject when you click `Add HTML file` and `Add CSS file`.  Probably they should match.

### LINKED ISSUE #2079: Switch to console.log() from alert() in default JavaScript file
When you click `Add JavaScript file` we give you this:

screen shot 2017-05-12 at 11 24 55 am

Now that we have a built-in `console` UI it would be nice to switch to that from `alert`.

### PR DESCRIPTION
**Pull request by flukeout**
Updated the text formatting in the comments as well.

Fixes #2079
Fixes #2078

### CODE REVIEW

File: public/resources/default-files/js.txt [hunk 1]
```diff
@@ -1,21 +1,19 @@
-/***************************
-This is JavaScript (JS), the programming language that powers the web (and this is a comment, which you can delete).
+/*
+This is JavaScript (JS) file.
```
Comment (humphd) [inline public/resources/default-files/js.txt]: Can you say `This is a JavaScript (JS) file.` here please? (i.e., add the indefinite article).

Comment (flukeout) [inline public/resources/default-files/js.txt]: Oh yeah, typo sorry.

**Comment by humphd** [comment]
One nit, and this is good to land.

**Comment by humphd** [comment]
This will also fix #2079

**Comment by flukeout** [comment]
Fixed nit and landing!

**Pull request merged**